html/admin/api.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
Yes, a reload was sufficient. Not sure whether you've done conditional reload/restarts before, but the idiomatic way is to create a file for php5-fpm like `roles/dns-dhcp/handlers/main.yml` and a notify statement in the task that modifies `/etc/sudoers`. The [ansible doco explains it](https://docs.ansible.com/ansible/playbooks_best_practices.html#task-and-handler-organization-for-a-role)